HeliosCLI/Makefile

```diff
@@ -34,6 +34,14 @@ ifneq ($(INCLUDES),)
     CFLAGS+=$(INCLUDES)
 endif
 
+# Architecture passed from parent Makefile (default to x64 if run standalone)
+ARCH ?= x64
+
+# Build and Output directories relative to this Makefile
+BUILD_DIR = $(ARCH)
+OUTPUT_DIR = output

```

Comment:
output must also be arch-specific, otherwise say you have x86/x64 you can't build them side by side. 

Either the binary name or the output folder needs to reflect the arch. 

You could do away with output/ entirely and just dump the binary in the arch directory with the object files, that's kinda fine too.